packages/ui/src/eth/bridge.spec.ts

```diff
@@ -0,0 +1,121 @@
+import { BigNumber, Wallet } from "ethers";
+import { mainnet, taiko } from "../domain/chain";
+import type { Bridge, BridgeOpts } from "../domain/bridge";
+import ETHBridge from "./bridge";
+
+const sendMessageTx = {
+  accessList: [undefined],
+  data: "0x123",
+  feeData: {
+    gasLimit: { hex: "0x01", type: "BigNumber" },
+    gasPrice: { hex: "0x01", type: "BigNumber" },
+    maxFeePerGas: { hex: "0x00", type: "BigNumber" },
+    maxPriorityFeePerGas: {
+      hex: "0x00",
+      type: "BigNumber",
+    },
+  },
+  from: "0x123",
+  network: 1,
+  nonce: 0,
+  to: "0x123",
+  hash: "0x123",
+  type: 1,
+  value: 100,
+};
+
+const mockSigner = {
+  getAddress: jest.fn(),
+};
+
+const mockContract = {
+  sendEther: jest.fn(),
+};
+
+jest.mock("ethers", () => ({
+  /* eslint-disable-next-line */
+  ...(jest.requireActual("ethers") as object),
+  Wallet: function () {
+    return mockSigner;
+  },
+  Signer: function () {
+    return mockSigner;
+  },
+  Contract: function () {
+    return mockContract;
+  },
+}));
+
+describe("bridge tests", () => {
```

Comment:
Seems the Codecov comment in this PR says current UI test coverage is 100%...? Is that correct?

Reply:
Yes, it is. It only looks for non-.svelte files (no UI testing).